src/environment.ts

```diff
@@ -72,7 +60,16 @@ export class SecureEnvironment implements MembraneBroker {
             throw ErrorCreate(`Missing SecureEnvironmentOptions options bag.`);
         }
         const { rawGlobalThis, secureGlobalThis, distortionMap } = options;
-        this.distortionMap = WeakMapCreate(isUndefined(distortionMap) ? [] : distortionMap.entries());
+        this.distortionMap = WeakMapCreate();
```

Comment:
we had some checks dispersed in different places to check the distortion when retrieved for usage, that had the side effect that only if used, you will see those errors. That check was also testing if a distortion is needed before extracting the distortion, this is why this is related to this PR, because that check was broken due to `document.all` issue.

Nevertheless, we now validate each and every one of the distortion entries during construction phase, resulting on early errors.